ddl/migrations/0175_add_user_score_tables.sql

```diff
@@ -0,0 +1,38 @@
+begin;
+
+-- Create index for plays by user ID and hourly timestamp
```

Comment:
nit: create the comment on the index itself!

Reply:
done